### ISSUE TITLE
npm v4.10 breaks build

### ISSUE THREAD

**Issue by gmaclennan** [digidem/mapeo-desktop#82]
The build script pre- f73ca76859ea0ebe4e6c796529f3e02ec866170c would install latest npm, which recently has been v4. Builds were broken because any dependencies that were listed as both dependencies and devDependencies were not included in the package. This did not happen when running locally (in dev environment), so likely related to production environment, and the pruning done by electron packager. The temporary fix is to lock to npm v3, but we should probably figure out where this bug is for future compatibility.

**Comment by noffle** [comment]
Which builds were breaking, @gmaclennan? [Travis](https://travis-ci.org/digidem/mapeo-desktop/builds) and [Appveyor](https://ci.appveyor.com/project/gmaclennan/ecuador-map-editor/history) going pretty far back. Can you point me to one that broke because of `npm@4`?

**Comment by gmaclennan** [comment]
The build didn't fail, but the build was broken. I only tested on Mac, but the v1.5.0 release fails because of a missing dep. The del was listed as both a dependency and a devDependency of OSM-p2p-server. I fixed that but then got a failure with v1.5.1 of another missing dep, this time a del that was listed as dependency and devDependency of a sub-dep. it seems like the pruning in npm v4 is overly aggressive. Or it could be a electron-packger issue? I wonder if there is a way to test a build actually opens / loads / installs on CI?
